_docs/.custom_wordlist.txt

```diff
@@ -5,6 +5,7 @@ args
 codebase
 databag
 databags
+datasource
```

Comment:
I feel like this should be "data source" in the description and so removed here.